deploy/examples/create-external-cluster-resources.py

```diff
@@ -990,65 +1249,85 @@ def validate_rgw_endpoint(self):
+            "client.csi-rbd-node"
+        )
+        self.out_map["CSI_RBD_PROVISIONER_SECRET"] = self.create_cephCSIKeyring_user(
+            "client.csi-rbd-provisioner"
+        )
+        self.out_map["CEPHFS_POOL_NAME"] = self._arg_parser.cephfs_data_pool_name
+        self.out_map[
+            "CEPHFS_METADATA_POOL_NAME"
+        ] = self._arg_parser.cephfs_metadata_pool_name
+        self.out_map["CEPHFS_FS_NAME"] = self._arg_parser.cephfs_filesystem_name
+        self.out_map[
+            "RESTRICTED_AUTH_PERMISSION"
+        ] = self._arg_parser.restricted_auth_permission
+        self.out_map["RADOS_NAMESPACE"] = self._arg_parser.rados_namespace
+        self.out_map["CSI_CEPHFS_NODE_SECRET"] = ""
+        self.out_map["CSI_CEPHFS_PROVISIONER_SECRET"] = ""
         # create CephFS node and provisioner keyring only when MDS exists
-        if self.out_map['CEPHFS_FS_NAME'] and self.out_map['CEPHFS_POOL_NAME']:
-            self.out_map['CSI_CEPHFS_NODE_SECRET'] = self.create_cephCSIKeyring_user(
-                "client.csi-cephfs-node")
-            self.out_map['CSI_CEPHFS_PROVISIONER_SECRET'] = self.create_cephCSIKeyring_user(
-                "client.csi-cephfs-provisioner")
-        self.out_map['RGW_ENDPOINT'] = self._arg_parser.rgw_endpoint
-        self.out_map['RGW_TLS_CERT'] = ''
-        self.out_map['MONITORING_ENDPOINT'], \
-            self.out_map['MONITORING_ENDPOINT_PORT'] = self.get_active_and_standby_mgrs()
-        self.out_map['RBD_POOL_NAME'] = self._arg_parser.rbd_data_pool_name
-        self.out_map['RBD_METADATA_EC_POOL_NAME'] = self.validate_rgw_metadata_ec_pool_name()
-        self.out_map['RGW_POOL_PREFIX'] = self._arg_parser.rgw_pool_prefix
+        if self.out_map["CEPHFS_FS_NAME"] and self.out_map["CEPHFS_POOL_NAME"]:
+            self.out_map["CSI_CEPHFS_NODE_SECRET"] = self.create_cephCSIKeyring_user(
+                "client.csi-cephfs-node"
+            )
+            self.out_map[
+                "CSI_CEPHFS_PROVISIONER_SECRET"
+            ] = self.create_cephCSIKeyring_user("client.csi-cephfs-provisioner")
+        self.out_map["RGW_ENDPOINT"] = self._arg_parser.rgw_endpoint
+        self.out_map["RGW_TLS_CERT"] = ""
+        (
+            self.out_map["MONITORING_ENDPOINT"],
+            self.out_map["MONITORING_ENDPOINT_PORT"],
+        ) = self.get_active_and_standby_mgrs()
+        self.out_map["RBD_POOL_NAME"] = self._arg_parser.rbd_data_pool_name
+        self.out_map[
+            "RBD_METADATA_EC_POOL_NAME"
+        ] = self.validate_rgw_metadata_ec_pool_name()
+        self.out_map["RGW_POOL_PREFIX"] = self._arg_parser.rgw_pool_prefix
         if self._arg_parser.rgw_endpoint:
             if self._arg_parser.dry_run:
                 self.create_rgw_admin_ops_user()
             else:
-                self.out_map['RGW_ADMIN_OPS_USER_ACCESS_KEY'], self.out_map['RGW_ADMIN_OPS_USER_SECRET_KEY'], info_cap_supported = self.create_rgw_admin_ops_user()
+                (
```

Comment:
Hey @subhamkrai I am not sure about this formatting(of using () and separate lines just for variables) and best practices but this type of formatting makes me more difficult to understand the code.

Reply:
can you point from the original file which lines you are pointing?
if you pointing to this line then I see what change tool has brought we can discuss offline also if you want.